.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
+go:
+- 1.13.x
```

Comment:
Could we indent arrays by two spaces (off the key under which they are). Understandably, this ... to my surprise, is still valid YAML. 🙄

This applies in other instances below e.g. `before_install` etc.

Reply:
Done https://github.com/Intellection/vault-init/pull/1/commits/2c564f7c5b31ca61115fe52902930c2ad978c958

Reply:
So far so good ... but I'm feeling guilty because I keep commenting of stuff that fundamentally doesn't matter, but there's still some more stuff you didn't indent. See `include:` and `before_deploy:`.

Reply:
Ah, missed it. Thank you. Put all the indents in one commit https://github.com/Intellection/vault-init/pull/1/commits/5228613499a350a3eb72c5563225abcf80fc2c2b